backend/config/settings.py

```diff
@@ -9,6 +9,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/5.2/ref/settings/
 """
+AUTH_USER_MODEL = 'users.UserProfile'
```

Comment:
It's bad practice to put variables before imports. Move AUTH_USER_MODEL below, after the environment initialization, so that the order is logical and predictable.

Recommended order:
```suggestion
import os
from pathlib import Path
import environ

BASE_DIR = Path(__file__).resolve().parent.parent

env = environ.Env()
env.read_env(os.path.join(BASE_DIR, '.env'))

SECRET_KEY = env("DJANGO_SECRET_KEY")
...

AUTH_USER_MODEL = 'users.UserProfile'

```

Reply:
Corrected.